config/jobs/kubernetes-sigs/image-builder/image-builder-presubmits.yaml

```diff
@@ -65,10 +65,9 @@ presubmits:
     annotations:
       testgrid-dashboards: sig-cluster-lifecycle-image-builder
       testgrid-tab-name: pr-json-sort-check
-  - name: packer-validate
+  - name: pull-packer-validate
```

Comment:
@codenrhoden do you mind removing run_if_changed from the gcp job while you're here?

Reply:
@CecileRobertMichon Added as a second commit.